site/_posts/2016-10-27-Announcing-Euwe-Release-Candidate.html.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Euwe Release Candidate
+author: chessbyte
+date: 2016-10-27 12:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Release Cadidate of ManageIQ Euwe!
+
+Euwe is the next milestone release for ManageIQ cloud and virtualization management platform. With each release, ManageIQ gets more robust and feature complete across providers. In this release, we concentrated on stablization fixes prior to the upcoming GA release. There are also updates to Providers as well as continuing performance improvements.
+
+* Microsoft SCVMM: Set default security protocol to ssl.
+* PostgreSQL High Availability:
+** Added repmgr to support automatic failover.
+** Maintain list of active standby database servers.
+** Added pg-dsn_parser for converting DSN to a hash.
+* Added Google pre and post provisioning methods.
+* Appliance Console: Removed menu items that are not applicable when running inside a container.
+* Performance: Services -> Workloads -> All VMs page load time reduced from 93,770ms to 524ms (99%) with a test of 20,000 VMs.
+
```

Comment:
I think this sounds a little bit too much like "developerese".  The announcement should probably be a bit more user focused.

See the announcement that I created for the Beta announcement for examples.

This time around I might use:

* **Schedule Automate tasks** through the user interface in the same way that you schedule SmartState Analyses.

* **Notification Methods**  have been added to Automate allow users to create custom notifications.

* **Chargeback Calculations** have been reworked based on interval duration instead of being based on the count of metrics. 

*   **Topology views** have been added to both OpenStack Cloud and OpenStack Infrastructure providers.

I believe we talked about some of the other items in the beta announcement.